### PR TITLE
Added web workers blog

### PR DESCRIPTION
**Pull request by vyasriday**

### CODE REVIEW

File: content/blog/adding-multi-threading-to-javascript-using-web-workers/index.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+entry(persons)
+```
+
+`worker.js`
+
+```javascript
+// addEventListener is directly accessible in worker file
+addEventListener("message", event => {
+  // extract person passed from main thread from event object
+  let person = e.data
```
Comment (aman-agrwl) [inline content/blog/adding-multi-threading-to-javascript-using-web-workers/index.md:116]: `event.data`

File: content/blog/adding-multi-threading-to-javascript-using-web-workers/index.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Club JavaScripters</title>
+  </head>
+  <body>
+    <button>Can you click me?</button>
+    <h2>Let's do the party. 👯‍♂️‍</h2>
+    <ul id="club"></ul>
```
Comment (aman-agrwl) [inline content/blog/adding-multi-threading-to-javascript-using-web-workers/index.md:44]: ` <ul id="root"></ul>`

File: content/blog/adding-multi-threading-to-javascript-using-web-workers/index.md [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+    listItem.textContent = name
+    // if a person is not registered, register them first
+    if (!isMember) {
+      // create a new worker thread
+      worker = new Worker("worker.js")
+      // pass data to worker thread
+      worker.postMessage(name)
+      // listen to any data passed from worker thread
+      worker.addEventListener("message", event => {
+        if (e.data) {
```
Comment (aman-agrwl) [inline content/blog/adding-multi-threading-to-javascript-using-web-workers/index.md:96]: `event.data`

**Comment by vyasriday** [comment]
@aman-agrwl  Updated the PR

File: content/blog/adding-multi-threading-to-javascript-using-web-workers/index.md (hunk 2 above)
Comment (vyasriday) [reply content/blog/adding-multi-threading-to-javascript-using-web-workers/index.md:44]: updated the rest if the code for id `club`

**Pull request merged**